packages/components/src/side_scroller/index.jsx

```diff
@@ -0,0 +1,140 @@
+import { useEffect, useRef, useState } from 'react';
+import clsx from 'clsx';
+import { size, eventCoordinates, stopEvent } from 'nebenan-helpers/lib/dom';
+import eventproxy from 'nebenan-helpers/lib/eventproxy';
+import styles from './index.module.scss';
+
+import Draggable from '../draggable';
+import Controls from './controls';
+import { DISABLE_SCROLL_DISTANCE } from '../constants/swipe';
+
+const SideScroller = ({ className: passedClassName, children, ...props }) => {
+  const [canScrollLeft, setCanScrollLeft] = useState(false);
+  const [canScrollRight, setCanScrollRight] = useState(false);
+  const [height, setHeight] = useState(0);
+
+  const containerRef = useRef(null);
```

Comment:
I think we initialise refs usually without any value - it seems to be working as well. 
I personally prefer to initialise it with value anyway. 

@peterhass any thoughts?

Reply:
❤️‍🔥 don't care 🔥